libs/client/API.lua

```diff
@@ -112,12 +112,18 @@ function API:__init(client)
 	self._mutexes = setmetatable({}, mutexMeta)
 end
 
-function API:authenticate(token)
-	self._headers = {
-		{'Authorization', token},
-		{'User-Agent', f('DiscordBot (%s, %s)', package.homepage, package.version)},
-	}
-	return self:getCurrentUser()
+function API:authenticate(token, isWebhookClient)
```

Comment:
This is not necessary. User-Agent can just be set on initialization. I can make this change when necessary.